Controllers/patchReview.controller.js

```diff
@@ -0,0 +1,10 @@
+const { editReview } = require("../Models/editReview.model");
+
+exports.patchReviews = (req, res, next) => {
+  const reviewData = req;
```

Comment:
ideally you should process the data from the request here, before passing it to the model